hw/ip/racl_ctrl/dv/env/racl_ctrl_env.core

```diff
@@ -0,0 +1,30 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:racl_ctrl_env:0.1"
+description: "RACL_CTRL DV UVM environment"
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:dv:cip_lib
+      - lowrisc:dv:racl_ctrl_ral
```

Comment:
Does that work? the RAL is is specialized thing isn't it?

Reply:
This should point at the virtual core (the thing you were asking about in your first comment). So I *think* this should be doing the right thing.